docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,367 @@
+
+Online Certificate Status Protocol (OCSP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the configuration of OCSP, there are small changes to apply.
+
+First we need to allow the use of the template "OCSPResponseSigning" by the server, to do so navigate to `Server Manager` > `Roles` > `Active Directory Certificates Services` > expand `YourServerName`, right click `Certificate template` and chose `New template to issue`, in the list select "OCSPResponseSigning".
+
+After the installation of OCSP we need to create a "Revocation Configuration".
+
+To create the "Revocation Configuration" navigate to `Server Manager` > `Roles` > `Active Directory Certificate Services` and expand `OnlineResponder`. Then expand `YourServerName`, right click on "Revocation Configuration", select `Add Revocation Configuration`, click `Next`, chose a name for your configuration and click `Next`. 
+
+Chose `Select a certificate for an existing entreprise CA`, click `Next`. Click `Browse` and find your entreprise CA in the list, select it, click `OK` and then `Next`. Chose `Automatically select a signing certificate`, make sure `Auto-Enroll for an OCSP signing certificate` is selected, then chose the default template of OCSP which is `OCSPResponseSigning` in the drop down list next to `Certificate Template:`. You need to add providers only if you whish to use some CRL in addition to OCSP.
+
+Once created, right click the revocation configuration and select `Edit properties`, go to the tab `Signing`, here you need to select `Enable NONCE extension support` then click `OK`.
+
+Verify that your OCSP is in the settings of the CA. Right click your CA, chose `Properties`. Navigate to the tab `Extension`, in the drop down list `Select extension` chose `Authority Information Access (AIA)`. Make sure that you have the following in the list of locations: `http://<fqdn of ndes server>/ocsp`, if you do not have it, add it via the button `Add...`. In this menu type the "http://" then insert "<ServerDNSName>" and type "/ocsp", the line you add should look like the following: `http://<ServerDNSName>/ocsp`, then validate by clicking `OK`. Also verify that `Include in the online certificate status protocol(OCSP) extension` is selected.
+
+RADIUS Certificate generation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the PKI of Microsft include that all your certificates will be deliver by the root CA of the PKI. The public key of this certificate is necessary on PacketFence. 
+
+As for RADIUS authentication we will need to generate a certificate for FreeRADIUS in PacketFence.
+
+To generate the RADIUS certificate the template `WebServer` will be used.
+
+The idea now is to create the request (.CSR), a private key from the PacketFence server and submit the CSR to the NDES server. Connect to PacketFence via ssh and type the following in the CLI to generate the private key:
+
```

Comment:
Just this line is enough to generate the csr and the private key
openssl req -new -newkey rsa:2048 -nodes -keyout server.key -out server.csr